database/pgDatabase.js

```diff
@@ -40,8 +55,10 @@ Sidebar.init({
     unique: true,
     primaryKey: true,
   },
-  fullLifetimeAccess: DataTypes.STRING,
-  accessTypes: DataTypes.STRING,
+  fullLifetimeAccess: {
+    type: DataTypes.BOOLEAN,
+    defaultValue: false,
+  },
```

Comment:
great way to reduce the complexity and size of your db schemas with a simple byte-sized representation.  Another great callout as you show off your skills to future employers